js/src/renderer.js

```diff
@@ -1,29 +1,29 @@
-import * as widgets  from '@jupyter-widgets/base';
+import * as widgets from '@jupyter-widgets/base';
 import Handsontable from 'handsontable';
-import {extend} from 'lodash';
-var version = require('../package.json').version
+import { extend } from 'lodash';
+var version = require('../package.json').version;
 var semver_range = '~' + version;
 
-
 var RendererModel = widgets.WidgetModel.extend({
-    defaults: function() {
+    defaults: function () {
         return extend(RendererModel.__super__.defaults.call(this), {
-            _model_name : 'RendererModel',
-            _model_module : 'ipysheet',
-            _model_module_version : semver_range,
+            _model_name: 'RendererModel',
+            _model_module: 'ipysheet',
+            _model_module_version: semver_range,
             code: ''
         });
     },
-    initialize: function() {
+    initialize: function () {
         RendererModel.__super__.initialize.apply(this, arguments);
         // we add Handsontable manually as extra argument to put it in the scope
-        this.fn = new Function('Handsontable', 'return (' + this.get('code') + ')')
-        Handsontable.renderers.registerRenderer(this.get('name'), this.fn(Handsontable))
+        Handsontable.renderers.registerRenderer(this.get('name'), this.fn(Handsontable));
```

Comment:
I'm not sure about this change, maybe you had a strong reason why using `new Function('Handsontable', 'return (' + this.get('code') + ')')`. But eslint was complaining about the `new Function` thing.

Reply:
Yeah, it does not work without the `new Function`, I'll see later how to fix the linter properly. I guess ignoring the lint error for now is fine.